src/Emu/Metadata/SupportFiles/FrontierLabs/LogFile.cs

```diff
@@ -128,16 +134,40 @@ public static LogFile GetLogFile(string log)
             return null;
         }
 
+        public static LocalDateTime ParseDate(string dateTime)
```

Comment:
May need to handle FL008 as I did in: 

https://github.com/QutEcoacoustics/emu/blob/8433f614d828385b0a431b9411eb90c2f716fdf7/src/Emu/Filenames/FilenameParser.cs#L162-L167

Reply:
I think we're okay here. Looking at the log file for the fixtures you posted affected by FL008, the time stamp in the log file looks like: "01/06/2019 02:00:00". Different date formats and here it's not affected by the error. Even if it were the space would be trimmed since the day segment comes first.